src/js/utils/__snapshots__/github-api.test.ts.snap

```diff
@@ -78,6 +78,13 @@ Object {
 `;
 
 exports[`./utils/github-api.ts should format the notification reason 12`] = `
+Object {
+  "description": undefined,
```

Comment:
Just realised that this is broken in the source file - shall we update the actual file https://github.com/manosim/gitify/blob/master/src/js/utils/github-api.ts#L48 to use `DESCRIPTIONS['CI_ACTIVITY']` which will remove that `undefined`.